src/components/axes/numericAxis.ts

```diff
@@ -194,12 +194,51 @@ export module Axis {
       var labelGroupTransform = "translate(" + labelGroupTransformX + ", " + labelGroupTransformY + ")";
       this._tickLabelContainer.attr("transform", labelGroupTransform);
 
+      this._showAllTickMarks();
```

Comment:
I would agree in cases where the method's functionality is somewhat complex, but in this case the code is pretty self-explanatory. Naming the method is nice, but now you have to go look in the file for where it's defined...